client/src/services/events.js

```diff
@@ -1,15 +1,16 @@
-import BaseService from "./base";
 
+import { makeRequest } from './helper';
 
-export default class EventService extends BaseService {
+const baseUri = '/api/events/'
```

Comment:
why we are using endpoint as baseUri?

Reply:
its actually base path, not base uri, will update variable name